client/context/viper.go

```diff
@@ -68,6 +72,36 @@ func defaultChainID() (string, error) {
 	return doc.ChainID, nil
 }
 
+// EnsureAccountExists - Make sure account exists
+func EnsureAccountExists(ctx CoreContext, name string) error {
+	keybase, err := keys.GetKeyBase()
+	if err != nil {
+		return err
+	}
+
+	if name == "" {
+		return errors.Errorf("must provide a from address name")
+	}
+
+	info, err := keybase.Get(name)
+	if err != nil {
+		return errors.Errorf("no key for: %s", name)
+	}
+
+	accAddr := sdk.AccAddress(info.GetPubKey().Address())
+
+	Acc, err := ctx.QueryStore(auth.AddressStoreKey(accAddr), ctx.AccountStore)
```

Comment:
(nit) why uppercase `Acc`?

Reply:
we should fix these things before merging, or at least open issues for follow up fixes ...